src/modules/data/schemas/Transaction.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "ITransaction",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "integer"
+    },
+    "inputs": {
+      "items": {
+        "type": "object"
+      },
+      "type": "array"
+    },
+    "outputs": {
+      "items": {
+        "type": "object"
+      },
+      "type": "array"
+    }
+  },
+  "additionalProperties": false,
+  "required": ["type", "inputs", "outputs"]
```

Comment:
What happens if there are no inputs like Genesis Block?

Reply:
That's a good question.
In the absence of the content, it consists of:
```
{
...
    "inputs": [],
...
}
```

Reply:
When validation, Does it not get caught on `required` items?

Reply:
The transactions in the Genesis block have no elements in the inputs.
The transaction is being handled normally.

Reply:
The test code below is also being processed normally.
``` TypeScript
        assert.ok(Validator.isValidOtherwiseThrow<ITransaction>
        ("Transaction",
        {
            "type": 0,
            "inputs": [],
            "outputs": [
                {
                    "value": "400000000000",
                    "address": "GA3DMXTREDC4AIUTHRFIXCKWKF7BDIXRWM2KLV74OPK2" +
                        "OKDM2VJ235GN"
                }
            ]
        }));
```

Reply:
The previous test code was also left.